contracts/strategies/QVSimple.sol

```diff
@@ -228,6 +224,24 @@ contract QVSimple is CoreBaseStrategy, RecipientsExtension {
         voiceCreditsAllocated[_sender] += voiceCreditsToAllocate;
     }
 
+    /// @notice Add allocator
+    /// @dev Only the pool manager(s) can call this function and emits an `AllocatorAdded` event
+    /// @param _allocator The allocator address
+    function _addAllocator(address _allocator) internal virtual {
+        allowedAllocators[_allocator] = true;
+
+        emit AllocatorAdded(_allocator, msg.sender);
+    }
+
+    /// @notice Remove allocator
+    /// @dev Only the pool manager(s) can call this function and emits an `AllocatorRemoved` event
+    /// @param _allocator The allocator address
+    function _removeAllocator(address _allocator) internal virtual {
+        allowedAllocators[_allocator] = false;
+
+        emit AllocatorRemoved(_allocator, msg.sender);
+    }
+
```

Comment:
This sequence is repeated in a lot of strats, initially I thought about an `AllocatorsExtension`. It should include `add/remove` functions along with the allocation time variables. wdty?